### PR TITLE
[T3A4][W15-A3]Sriram Sami

### PR DESCRIPTION
**Pull request by frizensami**

### CODE REVIEW

**Comment by frizensami** [comment]
Ready for review.

File: src/seedu/addressbook/Main.java [hunk 1]
```diff
@@ -107,7 +107,13 @@ private CommandResult executeCommand(Command command)  {
         try {
             command.setData(addressBook, lastShownList);
             CommandResult result = command.execute();
-            storage.save(addressBook);
+            
+            // Save the addressBook object onto disk and check if the storage file had to be (re)created
+            boolean isFileCreated = storage.save(addressBook);
+            if (isFileCreated) {
```
Comment (okkhoy) [inline src/seedu/addressbook/Main.java]: well, this is the right way to do this, but unfortunately, you are not using exception mechanism here; hence defeating the purpose of this exercise :-P

**Comment by okkhoy** [comment]
@frizensami 
Some comments added. Please ack & close the PR after reading comments.

**Comment by frizensami** [comment]
Acknowledged - I will create a whole other exception class for the next time :)

**Comment by okkhoy** [comment]
great! please close the PR

**Pull request closed without merging**